Automattic/wp-calypso
# Media Modal: Tone Down Accent Colours

#### Changes proposed in this Pull Request

This aligns the Media Library/Modal with @drw158's points here: https://github.com/Automattic/wp-calypso/issues/30026#issuecomment-453194803

#### Testing instructions

Visit your Media Library and verify that the "Add New" button is primary, but the edit button is not. Start writing a post and open your Media Library (using an image-type block if writing in Gutenberg) and verify that "Add New" is not primary, but "Insert" is. 

<img width="404" alt="Screenshot 2019-06-10 at 17 15 09" src="https://user-images.githubusercontent.com/43215253/59209491-54ca8780-8ba3-11e9-9cd5-92fdfac8391f.png">

cc @jsnajdr, @flootr 

Fixes #30026
